docs/README.md

```diff
@@ -0,0 +1,26 @@
+Table of Contents
```

Comment:
@Florence-Njeri @pratik2315   Why is everything under the main `/docs` directory? Remember, we are documenting the **tool**, Generator Tool; that means you should be adding your content under `docs/tools/generator`.  

If we kept your latest outline topic example, you'd end up with URLs such as...
- `docs/tools/generator/overview`
- `docs/tools/generator/installation-guide`
- `docs/tools/generator/asyncapi-spec-file`
- `docs/tools/generator/configuration-file`
- `docs/tools/generator/configuration-file#special-parameters`
- `docs/tools/generator/templates`
- `docs/tools/generator/file-templates`
- `docs/tools/generator/authoring-templates#special-file-names`
- etc...